ddtrace/config.py

```diff
@@ -0,0 +1,14 @@
+from .pin import Pin
+
+
+def get_from(obj):
```

Comment:
This is the main and only API, since it's a plain dictionary that makes everything easy.